app/Http/Controllers/MangaController.php

```diff
@@ -7,14 +7,23 @@
 
 class MangaController extends Controller
 {
+    function __contstruct()
+    {
+        $this->middleware('permission:manga-list');
+        $this->middleware('permission:manga-create', ['only' => ['create','store']]);
+        $this->middleware('permission:manga-edit', ['only' => ['edit', 'update']]);
+        $this->middleware('permission:manga-delete', ['only' => ['destroy']]);
+    }
     /**
      * Display a listing of the resource.
      *
      * @return \Illuminate\Http\Response
      */
     public function index()
     {
-        //
+        $manga = Manga::latest()->paginate(5);
+        return view('manga.index',compact('manga'))
+            ->with('i', (request()->input('page', 1) - 1) * 5);
     }
```

Comment:
Werken we met pagina's

Reply:
Blijkbaar, als het niks is dan gaat het weer weg.

Reply:
Okii